frontend/src/components/question-select-modal/question-select-modal.tsx

```diff
@@ -47,26 +49,23 @@ const QuestionSelect = (): ReactElement => {
   };
 
   const presentQuestions = () => {
-    const websocket: WebSocket = new WebSocket(url);
-
-    websocket.onopen = () => {
-      websocket.send(
-        JSON.stringify({
-          action: "createRoom",
-          courseId: state.courseId,
-        })
-      );
+    createSession(
```

Comment:
I get error: `"Missing paramters. courseId and questions all required"` when I click present. Also the `console.log` should be removed.